File: Edge of the Ocean/makeArrayConsecutive2.go
func makeArrayConsecutive2(statues []int) int {
    largest, smallest := 0, 20;
    for _, i := range statues {
        if (i > largest) {
            largest = i;
        }
        if (i < smallest) {
            smallest = i;
        }
    }
    return 1 + largest - smallest - len(statues);
}

File: CodeSignal/Edge of the Ocean/js - 3:5/makeArrayConsecutive2.js
const makeArrayConsecutive2 = statues => {
    let sorted = statues.slice().sort((a, b) => a - b), 
        count = 0;
    for (let i = sorted[0]; i < sorted[sorted.length - 1]; i++) {
        if (!sorted.includes(i)) count++;
    }
    return count;
};